src/bridge/command.rs

```diff
@@ -53,6 +53,8 @@ fn create_platform_shell_command(command: &str, args: &[&str]) -> Option<StdComm
         let mut result = StdCommand::new("wsl");
         result.args(["$SHELL", "-lc"]);
         result.arg(format!("{} {}", command, args.join(" ")));
+        #[cfg(windows)]
+        std::os::windows::process::CommandExt::creation_flags(&mut result, 0x0800_0000);
```

Comment:
Could you change this to use the constant from the winapi create, that we are already using?

https://docs.rs/winapi/latest/winapi/um/winbase/constant.CREATE_NO_WINDOW.html

Reply:
Fixed!